core/domain/wipeout_service.py

```diff
@@ -415,14 +415,10 @@ def _hard_delete_explorations_and_collections(pending_deletion_request):
         pending_deletion_request.user_id,
         pending_deletion_request.exploration_ids,
         force_deletion=True)
-    pending_deletion_request.exploration_ids = []
```

Comment:
Could you explain this change?

Reply:
Basically, the `delete_explorations` is idempotent, so we should keep the list of explorations so that when we repeat the deletion cron job it will try to delete all the models again. This is what broke during the testing, there was 503 error the first time the deletion was run and since we set `exploration_ids` to `[]` in the second run there were no IDs to delete.